nova/tests/unit/compute/test_compute.py

```diff
@@ -12675,6 +12769,26 @@ def fake_get_flavor_by_flavor_id(flavor_id, ctxt=None,
         self.assertRaises(exception.FlavorNotFound,
             self.compute_api.resize, self.context, instance, '4')
 
+    def test_cannot_resize_from_ephemeral_to_volume_backed_flavor(self):
+        instance = self._create_fake_instance_obj()
+        orig_get_flavor_by_flavor_id = \
+                flavors.get_flavor_by_flavor_id
+
+        def fake_get_flavor_by_flavor_id(flavor_id, ctxt=None,
+                                                read_deleted="yes"):
+            instance_type = orig_get_flavor_by_flavor_id(flavor_id,
+                                                                ctxt,
+                                                                read_deleted)
```

Comment:
These lines don't look like they're indented correctly. Is this only github?

Reply:
Yes, I think it is a Github issue. pep8 passes.